sky/setup_files/setup.py

```diff
@@ -97,11 +97,11 @@ def parse_readme(readme: str) -> str:
     'filelock>=3.6.0',
     # Adopted from ray's setup.py:
     # Tracking issue: https://github.com/ray-project/ray/issues/30984
-    "grpcio >= 1.32.0, <= 1.49.1; python_version < '3.10' and sys_platform == 'darwin'",  # noqa:E501
-    "grpcio >= 1.42.0, <= 1.49.1; python_version >= '3.10' and sys_platform == 'darwin'",  # noqa:E501
+    "grpcio >= 1.32.0, <= 1.49.1, != 1.48.0; python_version < '3.10' and sys_platform == 'darwin'",  # noqa:E501
```

Comment:
Should add a note / link on why this ver is excluded.

Reply:
Added. Thanks!